script/test/fixtures/volume-mounts/volumes-from/docker-compose.yml

```diff
@@ -1,20 +1,32 @@
-version: "2"
+version: '2'
 
 services:
-  web:
-    image: centos/httpd
+
+  afoo:
+    image: busybox
+    command: sleep 3600
     volumes:
-      - "./app:/src/app"
-    ports:
-      - "3030:3000"
-    command: nodemon -L app/bin/www
+       - /asdf
+       - /afoo
+       - /asuraj
+    volumes_from:
+       - cat
+
 
-  nginx:
-    restart: always
-    image: nginx
-    ports:
-      - "80:80"
+  bar:
+    image: busybox
+    command: sleep 3600
     volumes:
-      - /www/public
+       - /asdf
+       - /bar1
+       - /barsuraj
     volumes_from:
-      - web
+       - afoo
+
+  cat:
+    image: busybox
+    command: sleep 3600
+    volumes:
+      - /cat
+      - /catsuraj
+
```

Comment:
Please don't change the current test example.

Can you please take the above code and create a different, separate example from this?